docs/src/data/fetch-stars.ts

```diff
@@ -0,0 +1,17 @@
+export async function fetchStars() {
+  try {
+    const response = await fetch("https://api.github.com/repos/DiceDB/dice");
+    const data = await response.json();
+    const starCount = data.stargazers_count;
+
+    const formattedStars =
+      starCount >= 1000
+        ? (starCount / 1000).toFixed(1) + "k+"
+        : starCount.toString();
+
+    return formattedStars;
+  } catch (error) {
+    console.error("Failed to fetch GitHub stars:", error);
+    return null;
```

Comment:
this is a bad practice. We ideally should not have a published website with no stars count. So, this should break the build. Do not gulp the error.